scripts/fix_error_ids.py

```diff
@@ -102,38 +103,93 @@ def fix_ids(used_ids, file_names):
         fix_ids_in_file(file_name, available_ids, used_ids)
 
 
-def find_source_files(top_dir):
-    """Builds the list of .h and .cpp files in top_dir directory"""
+def find_files(top_dir, sub_dirs, extensions):
+    """Builds a list of files with ginen extensions in sub_dirs subdirectories of top_dir directory"""
```

Comment:
```suggestion
    """Builds a list of files with given extensions in sub_dirs (subdirectories) of top_dir directory"""
```

Reply:
Thanks! The typo is fixed and the docstring is slightly rephrased.